### PR TITLE
Adds MTU note for SDN to OVNK migration

### PR DESCRIPTION
**Pull request by stevsmit**
Additional info for modifying MTU prior to migration. 

Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OCPBUGS-27427

Link to docs preview:
https://70655--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn#nw-ovn-kubernetes-migration_migrate-from-openshift-sdn

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by weliang1** [comment]
@stevsmit OVN live migration will be supported in 4.15 release, please refer to the documents below for your release docs:
1.https://issues.redhat.com/browse/NP-41
2.https://github.com/openshift/enhancements/blob/master/enhancements/network/sdn-live-migration.md
3.https://docs.google.com/document/d/1gwEg2MZgyLb_P0wCH3haROvuHCRavBtijJxkUAKx1Bs/edit#heading=h.h3bqkn8gslkq

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -84,7 +84,10 @@ where:
 . Optional: You can customize the following settings for OVN-Kubernetes to meet your network infrastructure requirements:
 +
 --
-* Maximum transmission unit (MTU)
+* Maximum transmission unit (MTU). Consider the following prior to customizing the MTU for this optional step:
+** If you have the default MTU applied, and you want to keep the default MTU during migration, this step can be ignored. 
+** If you have a custom MTU applied, and you want to keep the custom MTU during migration, you must declare the value in this step.
```
Comment (weliang1) [inline modules/nw-ovn-kubernetes-migration.adoc:89]: @stevsmit How about updating it to say, 'You must declare the custom MTU value in this step'?

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 2]
```diff
@@ -84,7 +84,15 @@ where:
...
-* Maximum transmission unit (MTU)
+* Maximum transmission unit (MTU). Consider the following prior to customizing the MTU for this optional step:
+** If you have the default MTU applied, and you want to keep the default MTU during migration, this step can be ignored. 
+** If you have a custom MTU applied, and you want to keep the custom MTU during migration, you must declare the custom MTU value in this step.
+** This step does not work if you want to change the MTU value during migration. Instead, you must first follow the instructions for "Changing the cluster MTU". Then, you can keep the custom MTU value by performing this procedure and declaring the custom MTU value in this step. 
++
+[NOTE]
+====
+OpenShift-SDN and OVN-Kubernetes have different overlay overhead. MTU values should be selected by following the guidelines found on the "MTU value selection" page.
+====
```
Comment (agantony) [inline modules/nw-ovn-kubernetes-migration.adoc:95]:
```suggestion
* Maximum transmission unit (MTU). Consider the following before you adjust the MTU for this optional step:
** If you used the default MTU and want to keep the default MTU during the migration, you can ignore this step.
** If you used a custom MTU and want to keep the custom MTU during the migration, you must specify the custom MTU value in this step.
** This step does not work if you want to change the MTU value during the migration. Instead, you must first follow the instructions for "Changing the cluster MTU". You can then keep the custom MTU value by following this procedure and specifying the custom MTU value in this step.
+
[IMPORTANT]
====
OpenShift- SDN and OVN- Kubernetes have different overlay overhead. You need to select the MTU values according to the guidelines on the "MTU value selection" page.
====
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

**Pull request merged**